### PR TITLE
Change order of includes to allow grid-settings to work

### PR DESCRIPTION
**Pull request by cllns**
Unsure if we should change the comment on line 8 of `_base.scss`? 

I like keeping it as the original one, so users will have a hint if they mess things up.

### CODE REVIEW

File: source/assets/stylesheets/base/_base.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@
 @import "variables";
 
 // Neat Settings -- uncomment if using Neat -- must be imported before Neat
```
Comment (joshuaogle) [inline source/assets/stylesheets/base/_base.scss]: Let's either change the wording or remove this comment

**Comment by joshuaogle** [comment]
I'm in favor of turning this on by default, good idea. I don't think we should leave the comment as it is though, some people may not be as familiar with the default Neat setup.

Thanks @cllns!

**Comment by cllns** [comment]
That makes sense. Just changed the wording of that comment.

Thanks for open sourcing this! Using it for the first time right now. Makes it **so** quick to set-up a static site!

**Comment by joshuaogle** [comment]
Looks good. :+1: 
I'll get the other kits updated to do the same.

**Pull request merged**